Sortable.js

```diff
@@ -686,137 +686,134 @@
 				!options.dragoverBubble && evt.stopPropagation();
 			}
 
-			if (dragEl.animated) {
-				return;
-			}
+			// Smart auto-scrolling
+			_autoScroll(evt, options, this.el);
```

Comment:
Maybe it fixes the issue with not working scroll in Chrome?
https://github.com/RubaXa/Sortable/issues/935